mslib/msui/_tests/test_mscolab.py

```diff
@@ -156,26 +151,29 @@ def test_activate_project(self):
         assert self.window.mscolab.active_pid is not None
 
     def test_view_open(self):
-        pytest.skip("yet to refactor for new UI")
         self._connect_to_mscolab()
         self._login()
-        # test without activating project
-        QtTest.QTest.mouseClick(self.window.topview, QtCore.Qt.LeftButton)
-        QtTest.QTest.mouseClick(self.window.sideview, QtCore.Qt.LeftButton)
-        QtTest.QTest.mouseClick(self.window.tableview, QtCore.Qt.LeftButton)
-        QtWidgets.QApplication.processEvents()
-        assert len(self.window.active_windows) == 0
+        # # test without activating project
+        # QtTest.QTest.mouseClick(self.window.topview, QtCore.Qt.LeftButton)
```

Comment:
Since there is always an active waypoints model at any point, trying to open views before activating a project will just open a view for the local flighttrack. Only when the user activates a project, the waypoints model changes to the project's.